doc/Motr_Terminology_Guide.rst

```diff
@@ -0,0 +1,161 @@
+| User                                                    | Any Motr component or external application using a cas instance by sending fops to it.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
++---------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Value                                                   | An arbitrary sequence of bytes, associated with a key                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
++---------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
```

Comment:
Please add more:
Index,
DIX
RM
DTX
ADDB
BALLOC
BE
CAS
CM
CONSOLE
FI
HSM
RPC
State Machine
XCODE
UT
UB


Please refer to the doc/pdf
We have a lot of files. We can have more terminologies from there.

Reply:
Added the definitions.